<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1400</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>2000</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255)</string>
  </property>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>520</y>
     <width>1341</width>
     <height>250</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>250</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1341</width>
     <height>250</height>
    </size>
   </property>
   <property name="title">
    <string>장바구니</string>
   </property>
   <widget class="QTableWidget" name="Course_Basket">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>1320</width>
      <height>220</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1320</width>
      <height>220</height>
     </size>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::DragDrop</enum>
    </property>
    <property name="defaultDropAction">
     <enum>Qt::MoveAction</enum>
    </property>
    <property name="gridStyle">
     <enum>Qt::SolidLine</enum>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string>대학</string>
     </property>
     <property name="background">
      <color>
       <red>242</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </property>
    </column>
    <column>
     <property name="text">
      <string>개설학과</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>캠퍼스</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>학년</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>과정</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>이수구분</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>과목번호</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>과목명</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>학점-시간</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>담당교수</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>폐강여부</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>강의시간</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>비고</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupMust">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>650</width>
     <height>450</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>450</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>650</width>
     <height>500</height>
    </size>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="title">
    <string>꼭</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupPrefer">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>30</y>
     <width>641</width>
     <height>450</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>450</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>650</width>
     <height>450</height>
    </size>
   </property>
   <property name="title">
    <string>들으면 좋음</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Must_Remove">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>490</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>그룹 삭제</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Prefer_Remove">
   <property name="geometry">
    <rect>
     <x>1290</x>
     <y>490</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>그룹 삭제</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>0</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="QPushButton" name="Button_Create">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color : rgb(238, 238, 238)</string>
      </property>
      <property name="text">
       <string>시간표 생성</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="Button_Schedule">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color : rgb(238, 238, 238)</string>
      </property>
      <property name="text">
       <string>시간표</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="Button_Courses">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color : rgb(238, 238, 238)</string>
      </property>
      <property name="text">
       <string>강의 검색</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
